migration/20180207-2-recreate-materialized-view.sql

```diff
@@ -36,6 +38,7 @@ as
     licensepools.availability_time,
     licensepools.collection_id,
     customlistentries.list_id,
+    customlistentries.edition_id as list_edition_id,
```

Comment:
Changing this migration file means that someone who already deployed the development version with this migration won't have the update run automatically. I propose changing the date on the migration file whenever you change it, even if it's not in a release yet.